src/runtime/components/overlays/Popover.vue

```diff
@@ -8,17 +8,17 @@
       role="button"
       @mouseover="onMouseOver"
     >
-      <slot :open="open" :close="close">
+      <slot :open="headlessOpen" :close="close">
```

Comment:
Shouldn't it be `:open="(open !== undefined) ? open : headlessOpen"` for both slots?